aepp/exportDatasetToCloudStorage.py

```diff
@@ -0,0 +1,179 @@
+import flowservice
+import destinationinstanceservice
+import os
+import time
+from utils import Utils
+
+class ExportDatasetToCloudStorage:
+    global flow_conn
+    global dis_conn
+    global utils
+    global username
+    def __init__(self):
+        self.username = os.getlogin()
+        self.flow_conn = flowservice.FlowService()
+        self.dis_conn = destinationinstanceservice.DestinationInstanceService()
+        self.utils = Utils()
+
+    def createDataFlowIfNotExists(self, dataset_id, compression_type, data_format, export_path, on_schedule):
+        dataflow_id = self.utils.check_if_exist("Platform", "dataflow_id")
```

Comment:
question here, what happens if for whatever reason the dataflow id is incorrect here, could that ever happen

Reply:
dataflow should be saved to config file when a flow is created successfully. If people manually update the config file for an incorrect value, it will throw error in the steps below.